lib/logmsg/logmsg.c

```diff
@@ -524,9 +524,9 @@ log_msg_set_value(LogMessage *self, NVHandle handle, const gchar *value, gssize
   if (_log_name_value_updates(self))
     {
       msg_debug("Setting value",
-                evt_tag_printf("msg", "%p", self),
-                evt_tag_str("name", name),
-                evt_tag_printf("value", "%.*s", (gint) value_len, value));
+                evt_tag_str("macro", name),
```

Comment:
Why is macro better than name here? This is a name from a name-value pair.

Reply:
Yes, I know where is it came from, but I think the "name" is misleading in this case. I would change the reference from name-value (this is an internal association) to macro-value (this association is more message related)
Just compare old - new style.
 * old:
```
Setting value; msg='0x7fc5e0013e00', name='SOURCE', value='s_local'
```
 * new
```
Setting value; macro='SOURCE', value='s_local', msg='0x7f44a8013e00'
```

Reply:
My first instinct was to dislike the `macro` name, because these are key-value pairs, thus `key` might be even better. But there's a very good case for the `macro` name: our documentation refers to these things as such. It makes sense to follow that naming convention in log messages too.

I still don't like that they're called macros, but changing `name` to `macro` here makes our naming more consistent with the documentation, and that's a good thing.

Reply:
macro is NOT a good name in this context. Once upon a time, we've concluded what would be good names for everything, and we basically said that everything could become a "property", macros would be read-only properties whereas name-value pairs would become just properties.

But until that happens "macros" should not be used, as everywhere else we are referencing the concept of name-value pairs.

How they are referenced unfortunately is not uniform either, sometimes we use "name" in other cases we just use "value" to identify the name value-pair. In the context above, the tag "name" should be ok to understand the message, without introducing a very different way of capturing the same concept.

Introducing the "property" naming would be cool, but I am afraid that's a much larger project that has ties in documentation, configuration syntax and log messages. So large, that it might never happen. But please let's not introduce a new one.

Reply:
Ok. 
Thanks for the explanation. I will change it back to "name".